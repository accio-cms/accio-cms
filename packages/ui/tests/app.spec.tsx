import React = require('react')
import serializer from 'jest-emotion'
import { create } from 'react-test-renderer'

import App from '../src/app'

expect.addSnapshotSerializer(serializer)

test("App without color", () => {
  const msg = 'Poop'
  const tree1 = create(
    <App msg={msg} />
  ).toJSON()
  expect(tree1).toMatchSnapshot()
})

test("App with blue color", () => {
  const msg = 'Poop'
  const dataColor = 'blue'
  const tree1 = create(
    <App msg={msg} dataColor={dataColor} />
  ).toJSON()
  expect(tree1).toMatchSnapshot()
})
