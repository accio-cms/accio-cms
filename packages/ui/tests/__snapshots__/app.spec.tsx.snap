// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App with blue color 1`] = `
.emotion-0 {
  display: inline-block;
  color: blue;
}

<h1
  className="emotion-0"
>
  Poop
</h1>
`;

exports[`App without color 1`] = `
.emotion-0 {
  display: inline-block;
  color: red;
}

<h1
  className="emotion-0"
>
  Poop
</h1>
`;
